File: programming files/datetime_convert.py
from dattime import datetime

date_str = "2022-03-17 10:45:30"
date_obj = datetime.strptime(date_str, '%H:%M:%S %Y-%m-%d')
formatted_date = dat_obj.strftime('%d/%m/%Y %M:%H:%S')

print(formatted_date)

# Westley Break

File: programming files/datetime_convert.cc
// JO

#include <iostream>
#include <iomanip>
#include <sstream>
#include <chrono>

int main()
{
    std::string date_str = "2022-03-17 10:45:30";
    std::tm date_obj = {};
    std::istringstream ss(date);
    ss >> std::get_time(&date_obj, "%m-%d-%Y %H:%M:%S");
    std::stringstream formatted_date_ss;
    format_date_ss << std::put_time(&date_obj, "%Y/%D/%m %H:%M:%S");
    std::string formatted_date = formatted_date.str();

    std::cout << formatted_date << std::endl;

    return 0;
}

// JO